src/main/scripts/src/app/core/dto/document.ts

```diff
@@ -20,11 +20,14 @@
 
 export class Document {
 
-  public id: string;
-  public creationDate: string;
-  public createdBy: string;
+  public _id: string;
+  public creation_date: string;
+  public created_by: string;
```

Comment:
I wanted to remove the `constructor` and `toDto` method after backend changes, so that I wouldn't have to parse it.